pyrf/gui/plot_widget.py

```diff
@@ -237,8 +239,9 @@ def plot_changed(self, state, changed):
 
         if 'y_axis' in changed:
             b = self.window.blockSignals(True)
-            self.window.setYRange(state['y_axis'][0] , state['y_axis'][1], padding = 0)
-            self.persistence_window.setYRange(state['y_axis'][0] , state['y_axis'][1], padding = 0)
+            self.delayed_tick_update()
+            self.window.setYRange(min(state['y_axis']) , max(state['y_axis']), padding = 0)
+            self.persistence_window.setYRange(min(state['y_axis']) , max(state['y_axis']), padding = 0)
```

Comment:
how many elements does state['y_axis'] have? watch out for O(n) operations when O(1) ones will do.